gulpfile.js

```diff
@@ -0,0 +1,80 @@
+const gulp = require('gulp')
+const concat = require('gulp-concat')
+const minify = require('gulp-minify')
+const umd = require('gulp-umd')
+const merge = require('merge-stream')
+const sass = require('gulp-sass')(require('sass'))
+
+const compileSass = (source, output) => {
+  return merge(
+    gulp.src(source).pipe(
+      sass({
+        quietDeps: true,
+        outputStyle: 'compressed',
+        includePaths: ['frontend/**/*.scss', 'node_modules/govuk-frontend', 'node_modules/@ministryofjustice/frontend'],
+      })
+    ),
+    gulp.src([
+      'node_modules/jquery-ui-dist/jquery-ui.min.css',
+      'node_modules/accessible-autocomplete/dist/accessible-autocomplete.min.css',
+    ])
+  )
+    .pipe(concat(output))
+    .pipe(minify())
+    .pipe(gulp.dest('./assets/stylesheets'))
+}
+
+gulp.task(
+  'dist:sass',
+  () =>
+    compileSass('frontend/application.scss', 'application.css') &&
+    compileSass('frontend/application-ie8.scss', 'application-ie8.css')
+)
+
+gulp.task('dist:javascript', () =>
+  gulp
+    .src(['frontend/namespace.js', 'frontend/**/*.js', '!frontend/init.js'])
+    .pipe(concat('activities.js'))
+    .pipe(
+      umd({
+        exports: function () {
+          return 'ActivitiesFrontend'
+        },
+        namespace: function () {
+          return 'ActivitiesFrontend'
+        },
+      })
+    )
+    .pipe(gulp.dest('./assets/javascript'))
+)
+
+gulp.task('dist:assets', () => {
+  return gulp
+    .src(['node_modules/govuk-frontend/govuk/assets/**/*', 'node_modules/@ministryofjustice/frontend/moj/assets/**/*'])
+    .pipe(gulp.dest('./assets'))
+})
+
+gulp.task('dist:package', () =>
+  gulp
+    .src([
+      'node_modules/jquery/dist/jquery.js',
+      'node_modules/jquery-ui-dist/jquery-ui.js',
```

Comment:
It looks like everywhere JQuery is used it can be replaced with vanilla JS?

The only exception seems to be the datepicker, however that's to be removed and seems to have accessible issues? I believe the datepicker is also the only place JQuery UI is being used?

Is JQuery / JQuery UI needed?